components/Register.jsx

```diff
@@ -1,7 +1,49 @@
 import React from "react";
+import Image from "next/image";
 
 const Register = () => {
-  return <div>Register</div>;
+  return (
+    <div className="font-bold w-[45%] h-[60%] m-4">
+      <container className=" bg-acm-blue rounded-[10%] grid">
+        <div
+          style={{ paddingTop: "12%" }}
```

Comment:
Wondering if there is anyway to add this padding using tailwind to keep consistency

##### ex: pt-[12%]